library/core/src/ptr/non_null.rs

```diff
@@ -289,8 +293,10 @@ impl<T: ?Sized> NonNull<T> {
     ///
     /// This is a convenience for [`with_addr`][Self::with_addr], see that method for details.
     ///
+    /// For more details see the equivalent method on a raw pointer, [`pointer::map_addr`].
```

Comment:
Hm, now we have two different "for more details"? `pointer::map_addr` says the same thing that it says here, so I think this new sentence here isn't adding anything.

Reply:
I can remove the other "for more details", that one points to the `NonNull::with_addr`.